#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/makefile.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: 
## DP: Author: Jari Aalto <jari.aalto@cante.net>
## DP: 
## DP: 
## DP:     Makefile: change name. Use pkg-onfig
## DP:     
## DP:     Signed-off-by: Jari Aalto <jari.aalto@cante.net>

@DPATCH@

diff --git a/Makefile b/Makefile
index 219e871..7eb6a75 100644
--- a/Makefile
+++ b/Makefile
@@ -8,22 +8,19 @@ clean:
 	rm -f Shutdown
 
 install:
-	install wmShutdown /usr/local/bin
-	install wmShutdown.xpm /usr/share/pixmaps/
-	install -g root -o root Shutdown /usr/local/bin
-	chmod +s /sbin/shutdown
-	chmod +s /usr/local/bin/Shutdown
-	@echo Installed. Just type wmShutdown to use it
+	install -m 755 wmShutdown $(DESTDIR)/usr/bin/wmshutdown
+	install -m 644 wmShutdown.xpm $(DESTDIR)/usr/share/pixmaps/wmshutdown.xpm
+	install -m 755 Shutdown $(DESTDIR)/usr/bin/wmshutdown-run
 
 uninstall:
-	rm -f /usr/local/bin/wmShutdown
-	rm -f /usr/local/bin/Shutdown
+	rm -f /bin/wmShutdown
+	rm -f /bin/Shutdown
 	rm -f /usr/share/pixmaps/wmShutdown.xpm
 	@echo ":-( Uninstalled."
 
 dock: wmShutdown.c
-	$(CC) -c -o wmShutdown.o wmShutdown.c `gtk-config --cflags` 
-	$(CC) -o wmShutdown wmShutdown.o `gtk-config --libs` 
+	$(CC) -c -o wmShutdown.o wmShutdown.c `pkg-config --cflags`
+	$(CC) -o wmShutdown wmShutdown.o `pkg-config --libs`
 
 shutdown: shutdown.c
-	$(CC) -o Shutdown shutdown.c 
+	$(CC) -o Shutdown shutdown.c
